src/pool/srv_iv.c

```diff
@@ -626,6 +816,9 @@ pool_iv_update(void *ns, int class_id, struct pool_iv_entry *pool_iv,
 	key.class_id = class_id;
 	pool_key = (struct pool_iv_key *)key.key_buf;
 	pool_key->pik_entry_size = pool_iv_len;
+	uuid_copy(pool_key->pik_uuid, key_uuid);
+
+	D_DEBUG(DB_TRACE, "pool_iv %p\n", pool_iv);
```

Comment:
Was this debug statement left intentionally?

Reply:
oh, yes, I will remove it.